### DISCUSSION TITLE
Sodium thermdynamic information

### DISCUSSION THREAD

**Discussion by ryani42** [neams-th-coe/cardinal#615] [General]
Hi,

Unusual question but when running thermal hydraulics for liquid sodium, what is the source of the thermodynamic properties of liquid sodium? I.e. what database is used with cardinal to obtain the thermodynamic properties?
I know its not a typical question but any pointers would be greatly appreciated.

Thanks in advance

**Comment by aprilnovak** [comment]
Hi @ryani42, fluid properties in MOOSE can be obtained from a number of sources. There are a number of open-source fluid properties (`UserObjects`) in the [MOOSE fluid property module](https://mooseframework.inl.gov/modules/fluid_properties/index.html). If you have access to Sockeye, then there are also a different set of properties available in Cardinal (in the `./contrib/sodium` directory). Alternatively, you can always just define the properties in the input file directly.

The answer to "which one" is being used is going to depend on the particular input files you're running. If you point me to them, I can be more specific with the database info.
